.github/workflows/macos.yml

```diff
@@ -66,8 +66,107 @@ jobs:
+            "PUBLISH_NAME: ${PUBLISH_NAME}"
+          exit
+      - uses: actions/upload-artifact@v4
+        with:
+          name: ${{env.ARTIFACT_FILE}}
+          path: ${{env.ARTIFACT_PATH}}
+      - name: Finish build
+        run: |
+          date -u
+          zsh -c 'echo "Build completed in $(printf "%0.2f" $(($[$(date +%s)-$(cat build/stamp)]/$((60.))))) minutes"'
+
+      - name: Publish artifacts
+        uses: softprops/action-gh-release@v2
+        if: ${{github.ref_type == 'tag' || github.ref_name == 'dev'}}
+        with:
+          tag_name: nightly-github-actions
+          files: |
+            ${{env.ARTIFACT_PATH}}
+
+  armbuild:
+    needs: build
+    runs-on: macos-14
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: |
+           date -u
+           mkdir build
+           date +%s > build/stamp
+           brew uninstall --ignore-dependencies libtiff
+           brew install libtiff gtk+3 gtkmm3 gtk-mac-integration adwaita-icon-theme libsigc++@2 little-cms2 libiptcdata fftw lensfun expat pkgconfig llvm shared-mime-info exiv2 jpeg-xl libomp | tee -a depslog
+           date -u
+           echo "----====Pourage====----"
+           cat depslog | grep Pouring
+           zsh -c 'echo "Completed installation of dependencies in $(printf "%0.2f" $(($[$(date +%s)-$(cat build/stamp)]/$((60.))))) minutes"'
+      - name: Configure build system
+        env:
+          CMAKE_CXX_STANDARD: 11
+          PKG_CONFIG_PATH: /opt/homebrew/opt/libtiff/lib/pkgconfig:opt/homebrew/opt/libffi/lib/pkgconfig:/ope/homebrew/opt/expat/lib/pkgconfig
+          C_FLAGS: >
+            -arch arm64 -Xpreprocessor -fopenmp /opt/homebrew/opt/libomp/lib/libomp.dylib -I/opt/homebrew/opt/libomp/include -I/opt/homebrew/include -I/opt/homebrew/opt/gdk-pixbuf/include -I/opt/homebrew/opt/libiconv/include -I/opt/homebrew/opt/libxml2/include -I/opt/homebrew/opt/expat/include -I/opt/homebrew/opt/libtiff/include
+        run: |
+          # GITHUB_REF is the ref that triggered the build, like
+          # refs/heads/new-feature - the next line parses that to REF: the branch
+          # name only (new-feature)
+          export RAW_THERAPEE_VERSION=$(git describe --tags | cut -f1,2 -d'.')
+          export REF=${GITHUB_REF##*/}
+          export C_FLAGS=$(echo -e $C_FLAGS | tr -d '\n')
+          cd build && date -u && date +%s > configstamp
+          cmake \
+            -DCMAKE_BUILD_TYPE="Release" \
+            -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
+            -DCMAKE_EXE_LINKER_FLAGS="-L. -L/opt/homebrew/lib -Wl,-rpath -Wl,/opt/homebrew/lib -L/opt/homebrew/opt/gdk-pixbuf/lib -L/opt/homebrew/opt/libiconv/lib -L/opt/homebrew/opt/libomp/lib -L/opt/homebrew/opt/libffi/lib -L/opt/homebrew/opt/libffi/lib -L/opt/homebrew/opt/libxml2/lib -L/opt/homebrew/opt/expat/lib" \
+            -DCACHE_NAME_SUFFIX="${RAW_THERAPEE_VERSION}-${REF}" \
+            -DPROC_TARGET_NUMBER="11" \
```

Comment:
What is target number 11? If it is for no specific target, "0" can be used, or just omit it because "0" is the default.

Reply:
@Lawrence37 Thank you.  I've also adjusted the patch so libraw_r works now,